hello-ethernet/Kconfig

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Golioth, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Golioth application options"
+
+if WIFI_ESP32
```

Comment:
Can we delete this section?

Reply:
Yes, I've removed it with the latest force-push.